src/components/Admin/Community/ApplicationPage.tsx

```diff
@@ -1,12 +1,225 @@
+    email: x.user.email,
+    state: x.lifecycle?.state || '',
+  }));
+
+interface ApplicationPageProps extends PageProps {
+  applications: ApplicationInfoFragment[];
+}
+
+export const ApplicationPage: FC<ApplicationPageProps> = ({ paths, applications }) => {
+  const { t } = useTranslation();
+
+  const [appChosen, setAppChosen] = useState<ApplicationInfoFragment | undefined>(undefined);
 
-export const ApplicationPage: FC<ApplicationPageProps> = ({ paths }) => {
   const currentPaths = useMemo(() => [...paths, { name: 'applications', real: false }], [paths]);
   useUpdateNavigation({ currentPaths });
-  return <div>Comming soon!</div>;
+
+  const applicationsVm = useMemo(() => toApplicationViewmodel(applications), [applications]);
+
+  const handleError = useApolloErrorHandler();
+
+  const [updateApplication] = useEventOnApplicationMutation({
+    onError: handleError,
+  });
+
+  const setNewStateHandler = (appId: string, newState: string) => {
+    updateApplication({
+      variables: {
+        input: {
+          applicationID: appId,
+          eventName: newState,
+        },
+      },
+    });
+  };
+
+  return (
+    <>
+      <div style={{ height: 400 }}>
+        <DataGrid
+          disableSelectionOnClick
+          rows={applicationsVm}
+          columns={colDef(t)}
+          density="compact"
+          hideFooter={true}
+          disableColumnSelector={true}
+          onRowClick={param => {
+            const appId = param.getValue(param.id, 'id');
+            setAppChosen(applications.find(x => x.id === appId));
+          }}
+        />
+      </div>
+      <ApplicationModal
+        appChosen={appChosen}
+        onHide={() => setAppChosen(undefined)}
+        onSetNewState={setNewStateHandler}
+      />
+    </>
+  );
 };
 export default ApplicationPage;
+
+const appStyles = createStyles(theme => ({
+  title: {
+    flexGrow: 1,
+  },
+  header: {
+    display: 'flex',
+    gap: theme.shape.spacing(4),
+    alignItems: 'center',
+    justifyContent: 'center',
+
+    [theme.media.down('sm')]: {
+      flexWrap: 'wrap',
+      gap: theme.shape.spacing(2),
+    },
+  },
+  profile: {
+    display: 'flex',
+    alignItems: 'center',
+    gap: theme.shape.spacing(1),
+
+    [theme.media.down('sm')]: {
+      gap: 0,
+      flexGrow: 1,
+    },
+  },
+  userName: {
+    whiteSpace: 'nowrap',
+    display: 'flex',
+
+    [theme.media.down('sm')]: {
+      flexGrow: 1,
+      justifyContent: 'center',
+    },
+  },
+  body: {
+    display: 'flex',
+    flexDirection: 'column',
+    maxHeight: 400,
+    overflowY: 'auto',
+  },
+  questions: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.shape.spacing(1),
+    margin: theme.shape.spacing(1),
+  },
+  question: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+}));
+
+interface ApplicationModalProps {
+  appChosen?: ApplicationInfoFragment;
+  onHide: () => void;
+  onSetNewState: (appId: string, newState: string) => void;
```

Comment:
`onChangeState` sounds better. `New` and `Old` aren't the best words to use in names.